spec/features/shelters/show_spec.rb

```diff
@@ -47,14 +47,13 @@
       expect(page).to have_content(review_1.title)
       expect(page).to have_content(review_1.rating)
       expect(page).to have_content(review_1.content)
-      # expect(page).to have_css("img[src*='image.jpeg']")
     end
 
     within ".review-#{review_2.id}" do
       expect(page).to have_content(review_2.title)
       expect(page).to have_content(review_2.rating)
       expect(page).to have_content(review_2.content)
-      expect(page).to_not have_css("img[src*='image.jpeg']")
+      expect(page).to_not have_css("img[src*='https://www.netclipart.com/pp/m/5-50989_gemini-thumbs-up-cube-kids-thumbs-up-clipart.png']")
```

Comment:
thanks for fixing this

Reply:
I did this after a message about an asset not present in asset pipeline, and it seemed to work, so I left it like that, but I definitely did not make changes to all of them.